README.md

```diff
@@ -91,5 +91,5 @@ Please check [CONTRIBUTING.md](CONTRIBUTING.md) and the [Developer Guide](docs/d
 
 ## Community
 
-* Join our [Kubeflow Slack Channel](https://kubeflow.slack.com/archives/C06627U3XU3)
+* Join our [Kubeflow Slack Channel](https://cloud-native.slack.com/archives/C074588U7EG)
```

Comment:
Please can you link this page: https://www.kubeflow.org/docs/about/community/#kubeflow-slack-channels ?
Since folks need to join CNCF Slack before joining the #kubeflow-spark-operator channel.

Reply:
on to it... will update by my EOD today

Reply:
@andreyvelich  @yuchaoran2011 I have incorporated both the points in PR.